src/android/StatusBar.java

```diff
@@ -164,4 +181,21 @@ private void setStatusBarBackgroundColor(final String colorPref) {
             }
         }
     }
+
+    private void setStatusBarTransparent(final boolean transparent) {
+        if (Build.VERSION.SDK_INT >= 21) {
+            final Window window = cordova.getActivity().getWindow();
+            if (transparent) {
+                window.getDecorView().setSystemUiVisibility(
+                        View.SYSTEM_UI_FLAG_LAYOUT_STABLE
+                                | View.SYSTEM_UI_FLAG_LAYOUT_FULLSCREEN);
+                window.setStatusBarColor(Color.TRANSPARENT);
```

Comment:
This method was only introduced in API level 21 so you should use the same approach as `setStatusBarBackgroundColor` and call it via reflection. In fact you'd be better off refactoring `setStatusBarBackgroundColor` so you don't end up repeating a bunch of code.